README.md

```diff
@@ -1,6 +1,8 @@
 # metal-project-action
 
-Experimental Github Action for creating [Equinix Metal](https://metal.equinix.com) Projects.
+[![Experimental](https://img.shields.io/badge/Stability-Experimental-red.svg)](experimental-statement.md#experimental-statement)
```

Comment:
I think we'll have an easier time linking out to the equinix-labs experimental-statement and making changes there. It is a file prone to change and the we'll want that same sentiment expressed in every project that uses the experimental badge.